docs/modules/ROOT/pages/documentation/microprofile/config/directory.adoc

```diff
@@ -0,0 +1,181 @@
+= Directory Config Source
```

Comment:
This new page should also be added to the [nav.adoc](https://github.com/payara/Payara-Community-Documentation/blob/master/docs/modules/ROOT/nav.adoc) file, otherwise it won't appear in the page tree in the left sidebar

Reply:
Done in 559f7e9. Thanks for pointing out :+1: